src/Monitor/Monitor/ActionGroups/NewAzureRmTestNotificationCommand.cs

```diff
@@ -0,0 +1,240 @@
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+namespace Microsoft.Azure.Commands.Insights.ActionGroups
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Threading;
+    using System.Linq;
+    using System.Management.Automation;
+
+    using Microsoft.Azure.Management.Monitor;
+    using Microsoft.Azure.Management.Monitor.Models;
+    using Microsoft.Azure.Commands.Insights.OutputClasses;
+    using Microsoft.Azure.Commands.Insights.TransitionalClasses;
+    using Newtonsoft.Json;
+    using ResourceManager.Common.ArgumentCompleters;
+
+    /// <summary> Create new test notifications </summary>
+    [Cmdlet("Test", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "ActionGroup", DefaultParameterSetName = ByPropertyName, SupportsShouldProcess = true)]
+    [OutputType(typeof(PSTestNotificationDetailsResponse))]
+    public class NewAzureRmTestNotificationCommand : ManagementCmdletBase
+    {
+        private const string ByPropertyName = "ByPropertyName";
+        private const string CompleteState = "Complete";
+        private readonly int MaxRetryCount = 30;
+        private readonly int WaitDurationBetweenRetryInSeconds = 10;
+
+        #region Cmdlet parameters
+
+        /// <summary>
+        /// Gets or sets the action group name parameter.
+        /// </summary>
+        [Parameter(ParameterSetName = ByPropertyName, Mandatory = true, HelpMessage = "The required alert type name")]
+        [ValidateNotNullOrEmpty]
+        public string AlertType { get; set; }
+
+        /// <summary>
+        /// Gets or sets the resource group parameter.
+        /// </summary>
+        [Parameter(ParameterSetName = ByPropertyName, Mandatory = true, HelpMessage = "The optional resource group name")]
+        [ResourceGroupCompleter]
+        [ValidateNotNullOrEmpty]
+        public string ResourceGroupName { get; set; }
+
+        /// <summary>
+        /// Gets or sets the action group name parameter.
+        /// </summary>
+        [Parameter(ParameterSetName = ByPropertyName, Mandatory = true, HelpMessage = "The optional action group name")]
+        [ValidateNotNullOrEmpty]
+        public string ActionGroupName { get; set; }
+
+        /// <summary>
+        /// Gets or sets the list of email receivers.
+        /// </summary>
+        [Parameter(ParameterSetName = ByPropertyName, Mandatory = true, HelpMessage = "The list of receivers")]
+        public List<PSActionGroupReceiverBase> Receivers { get; set; }
```

Comment:
please use "Receiver" instead of "Receivers", regenerate help markdown after renaming this parameter